### PR TITLE
Add monitoring section to PR template

### PR DESCRIPTION
**Pull request by sliptype**
### Description

As discussed, adding a section about monitoring/alerting to the PR template.

If this one looks good to you guys I'll put add it to `audius-protocol`

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 ### Dragons
 
 Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?
 
-
 ### How Has This Been Tested?
 
 Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
 
+### Is any new monitoring/alerting being added along with this change?
```
Comment (raymondjacobson) [inline .github/pull_request_template.md:11]: how about just

`### How will this change be monitored?`

**Review by raymondjacobson — APPROVED** [review]
lgtm! just a suggestion

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/sk-pr-template-add-monitoring

File: .github/pull_request_template.md (hunk 1 above)
Comment (sliptype) [reply .github/pull_request_template.md]: Much better! Done

**Pull request merged**